### DISCUSSION TITLE
.tooltip:has() -> Empty sub-selector hmm should I be concern on this.

### DISCUSSION THREAD

**Discussion by TomieAi** [saadeghi/daisyui#2852] [Q&A] [answered]
![image](https://github.com/saadeghi/daisyui/assets/111332073/2be9dc84-dd92-4faa-9803-a8b4f451042a)

I mean it works still. this when i'm using angular btw.

**Comment by saadeghi** [comment, marked as answer]
If the tooltips are working, there's nothing to worry about. The warning can be a result of CSS processing and minification.
